WORKSPACE

```diff
@@ -13,20 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-http_archive(
-    name = "rules_python",
-    url = "https://github.com/bazelbuild/rules_python/releases/download/0.0.2/rules_python-0.0.2.tar.gz",
-    strip_prefix = "rules_python-0.0.2",
-    sha256 = "b5668cde8bb6e3515057ef465a35ad712214962f0b3a314e551204266c7be90c",
-)
+workspace(name = "bazel_rules_fuzzing")
```

Comment:
From the docs, this should be "rules_fuzzing": https://docs.bazel.build/versions/master/skylark/deploying.html#workspace

Reply:
For the first time I know we belong to bazelbuild 😆